### ISSUE TITLE
Question: Are shortcuts to sharepoint mount points supported?

### ISSUE THREAD

**Issue by cyberatz** [jstaf/onedriver#104]
On Sharepoint sites, you have the ability to create a link to the folder repository in your OneDrive folder. 

I'm yet to find a tool (other than OneDrive) that supports these shortcuts / links to sharepoint mount points.

Before going down the path of getting one driver running, I thought I'd ask first :)

Thanks

**Comment by abraunegg** [comment]
@cyberatz 
The issue here is that the OneDrive API itself does not support the links. Refer to the following for more details: https://github.com/OneDrive/onedrive-api-docs/issues/1427

Will let @jstaf respond to this as well, but based on my investigations, the API does not handle / provide details about any shortcuts that are created.

**Comment by jstaf** [comment]
Yeah, I remember looking into this and the OneDrive API doesn't have a concept of shortcuts/symbolic links yet https://github.com/jstaf/onedriver/issues/80#issuecomment-639875511. I believe that Sharepoint sites might actually get treated as separate OneDrive drives in which case we might be able to add in links somewhere if this is the case (since accessing different drives wouldn't actually rely on the link functionality). 

Anyhow, in either case (normal symbolic links or Sharepoint sites as drives), it won't work yet :disappointed:. If anything, check back here once onedriver supports shared items (since that relies on the same multiple-drive functionality that multiple Sharepoint sites would).

**Comment by jeroengvv** [comment]
On Fedora the onedrive terminal tool does support links and shared_folders. Thus it should be possible.

**Comment by abraunegg** [comment]
@jeroengvv 
> On Fedora the onedrive terminal tool does support links and shared_folders. Thus it should be possible.

Whilst 'onedrive' supports SharePoint and OneDrive Business Shared Folders through specific drive_id configuration, it does *not* support 'links' created by the OneDrive 'Add Shortcut to My Files'. This is because the OneDrive API itself cannot handle these items. See:
* https://github.com/abraunegg/onedrive/issues/1224
* https://github.com/OneDrive/onedrive-api-docs/issues/1427
